packages/create-medusa-app/src/index.ts

```diff
@@ -126,23 +116,13 @@ export const run = async (): Promise<void> => {
       verbose: progOptions.verbose,
     })
   }
-  await newStarter({
-    starter: "https://github.com/medusajs/admin",
-    root: path.join(projectRoot, `admin`),
-    keepGit: true,
-    verbose: progOptions.verbose,
-  })
 
   console.log(`
   Your project is ready 🚀. The available commands are:
   
     Medusa API
     cd ${projectRoot}/backend
     yarn start
-
-    Admin
-    cd ${projectRoot}/admin
-    yarn start
   `)
 
   if (hasStorefront) {
```

Comment:
we should change what we print if the Next starter is included:

```
 Storefront
 cd ${projectRoot}/storefront
 yarn dev
```
Running `yarn start` will throw an error as that requires that you build the starter before running the command.

Reply:
Got it, will include in this PR 👍